tests/vmi_networking_test.go

```diff
@@ -817,7 +817,13 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				var err error
 
 				By("Create masquerade VMI")
-				networkData, err := libnet.CreateDefaultCloudInitNetworkData()
```

Comment:
I would keep the default factory there for now. If we will start using cloud init on other tests, it will be a shame not to re-use the same factory.
But even if we are to move it (I do not have a hard opinion on this), it better be on a separate PR/commit with a proper reasoning.

Reply:
Done